### PR TITLE
Unit test coverage

### PR DESCRIPTION
**Pull request by hanars**
remove unused and unreachable code and add tests for edge cases

### CODE REVIEW

File: seqr/views/utils/variant_utils.py [hunk 1]
```diff
@@ -60,6 +61,21 @@ def get_variant_key(xpos=None, ref=None, alt=None, genomeVersion=None, **kwargs)
     return '{}-{}-{}_{}'.format(xpos, ref, alt, genomeVersion)
 
 
+def saved_variant_genes(variants):
```
Comment (hanars) [inline seqr/views/utils/variant_utils.py:64]: This is copy-pasted from the api file into the utility file since it is shared in multiple places. Original here: https://github.com/macarthur-lab/seqr/pull/1352/files#diff-f34dfe2f07218b25094615c1bb2c39cbL350

**Comment by hanars** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 2
           

Complexity increasing per file
==============================
- seqr/views/utils/export_utils_tests.py  2
- reference_data/management/tests/update_omim_2_3_tests.py  1
         

Complexity decreasing per file
==============================
+ reference_data/management/commands/update_all_reference_data.py  -1
+ seqr/views/utils/pedigree_image_utils_tests.py  -1
+ seqr/views/apis/dataset_api.py  -1
+ seqr/views/apis/awesomebar_api.py  -1
+ reference_data/management/commands/utils/update_utils.py  -3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/hanars/seqr/pullRequest?prid=5739135&bid=18772782)

File: seqr/views/utils/pedigree_image_utils_tests.py [hunk 2]
```diff
@@ -2,73 +2,80 @@
...
 class PedigreeImageTest(TestCase):
     fixtures = ['users', '1kg_project']
 
-    def test_get_parsed_individuals(self):
+    @mock.patch('seqr.views.utils.pedigree_image_utils.BASE_DIR', '/')
+    @mock.patch('seqr.views.utils.pedigree_image_utils.random.randint')
+    @mock.patch('seqr.views.utils.pedigree_image_utils.os.system')
+    @mock.patch('seqr.views.utils.pedigree_image_utils.tempfile')
+    def test_update_pedigree_images(self, mock_tempfile, mock_os_system, mock_randint):
+        mock_tempfile.gettempdir.return_value = '/tmp'
```
Comment (hanars) [inline seqr/views/utils/pedigree_image_utils_tests.py:20]: Codacy found an issue: [Probable insecure usage of temp file/directory.](https://app.codacy.com/manual/hanars/seqr/pullRequest?prid=5739135)

File: seqr/views/apis/variant_search_api_tests.py [hunk 3]
```diff
@@ -10,7 +10,7 @@
 from seqr.views.apis.variant_search_api import query_variants_handler, query_single_variant_handler, \
     export_variants_handler, search_context_handler, get_saved_search_handler, create_saved_search_handler, \
     update_saved_search_handler, delete_saved_search_handler, get_variant_gene_breakdown
-from seqr.views.utils.test_utils import AuthenticationTestCase, VARIANTS
+from seqr.views.utils.test_utils import AuthenticationTestCase, VARIANTS, SINGLE_VARIANT
```
Comment (hanars) [inline seqr/views/apis/variant_search_api_tests.py:13]: Codacy found an issue: [Unused SINGLE_VARIANT imported from seqr.views.utils.test_utils](https://app.codacy.com/manual/hanars/seqr/pullRequest?prid=5739135)

**Comment by hanars** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- reference_data/management/tests/update_omim_2_3_tests.py  1
- seqr/views/utils/export_utils_tests.py  2
         

Complexity decreasing per file
==============================
+ seqr/views/utils/pedigree_image_utils_tests.py  -1
+ reference_data/management/commands/update_all_reference_data.py  -1
+ seqr/views/apis/awesomebar_api.py  -1
+ reference_data/management/commands/utils/update_utils.py  -3
+ seqr/views/apis/dataset_api.py  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/hanars/seqr/pullRequest?prid=5739135&bid=18772782)

File: seqr/views/utils/pedigree_image_utils_tests.py (hunk 2 above)
Comment (hanars) [inline seqr/views/utils/pedigree_image_utils_tests.py:20]: Codacy found an issue: [Probable insecure usage of temp file/directory.](https://app.codacy.com/manual/hanars/seqr/pullRequest?prid=5739135)

**Comment by hanars** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           

Complexity increasing per file
==============================
- seqr/views/utils/export_utils_tests.py  2
- reference_data/management/tests/update_omim_2_3_tests.py  1
         

Complexity decreasing per file
==============================
+ seqr/views/apis/dataset_api.py  -1
+ reference_data/management/commands/update_all_reference_data.py  -1
+ seqr/views/apis/awesomebar_api.py  -1
+ reference_data/management/commands/utils/update_utils.py  -3
+ seqr/views/utils/pedigree_image_utils_tests.py  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/hanars/seqr/pullRequest?prid=5739135&bid=18772782)

File: seqr/views/utils/pedigree_image_utils_tests.py (hunk 2 above)
Comment (hanars) [reply seqr/views/utils/pedigree_image_utils_tests.py:20]: this is okay to ignore
